cost/flexera/cco/cheaper_regions/README.md

```diff
@@ -1,13 +1,28 @@
 # Cheaper Regions Policy
 
-## What it does
+## What It Does
 
 This Policy Template determines which regions have cheaper alternatives by specifying the expensive region name and the cheaper region name for analysis
 
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Billing Center Name* - The list of Billing centers to check against
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
```

Comment:
This label is too long and doesn't line up with other policies. It should be renamed to just "Email Addresses"